src/components/application_manager/test/hmi_capabilities_test.cc

```diff
@@ -1125,6 +1125,41 @@ TEST_F(HMICapabilitiesTest,
   EXPECT_FALSE(file_system::FileExists(kHmiCapabilitiesCacheFile));
 }
 
+TEST_F(
+    HMICapabilitiesTest,
+    OnSoftwareVersionReceived_CcpuMatchNoPendingRequestsForCapability_SetHMICooperatingToTrue) {
+  // If local ccpu_version matches with received ccpu_version from HMI, and
+  // cache exist SDL Core should respond all pending RAI requests, this is
+  // possible if set the hmi_cooperating to true
+  const std::string ccpu_version = "4.1.3.B_EB355B";
+
+  hmi_capabilities_->set_ccpu_version(ccpu_version);
+
+  EXPECT_EQ(ccpu_version, hmi_capabilities_->ccpu_version());
+  EXPECT_CALL(mock_app_mngr_, SetHMICooperating(true));
+
+  hmi_capabilities_->OnSoftwareVersionReceived(ccpu_version);
+}
+
+TEST_F(
+    HMICapabilitiesTest,
+    OnSoftwareVersionReceived_CcpuMatchHavePendingRequestsForCapability_NoSetHMICooperatingToTrue) {
+  // If local ccpu_version matches with received ccpu_version from HMI, and
+  // cache does't exist, SDL Core should not respond all holded RAI requests,
+  // before will be processed required requests for capabilitiesit's. It's mean
+  // that hmi_cooperating need to be false until required requests will be
+  // proccesed.
+  const std::string ccpu_version = "4.1.3.B_EB355B";
+
+  hmi_capabilities_->Init(last_state_wrapper_);
```

Comment:
@ydementieiev 
// `Init()` is required to set pending capabilities requests to the HMI